common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1518,6 +1519,11 @@ public enum API {
          */
         public static final String GET_SSO_TOKEN_PATH = "/ssoToken";
 
+        /**
+         * Broker api path constant for execute device registration protocols.
+         */
+        public static final String DEVICE_REGISTRATION_PROTOCOLS_PATH = "/deviceRegistration/protocols";
```

Comment:
So every protocol will go through this path?

Reply:
yep